yml/roles/alfresco/handlers/main.yml

```diff
@@ -11,7 +11,7 @@
 - name: Clean dd temp perf files (Repository)
   file:
     state: absent
-    path: "{{ contentstore_directory | default(contentstore_dir) }}/.hc_tmp_dd_perf_file.raw"
+    path: "{{ contentstore_directory | default(contentstore_dir.stdout_lines[0] | regex_replace('\\${dir.root}',dir_root.stdout_lines[0])) }}/.hc_tmp_dd_perf_file_{{ inventory_hostname }}.raw"
```

Comment:
any reason for using double backslash? The rest of the playbook seems to use a single escaped backslash

Reply:
It is required because it is inside a double quoted string